styles.css.scss

```diff
@@ -0,0 +1,59 @@
+.navbar {
+  color: white;
+  background-color: #2B78E4;
+  .navbar-brand {
+    font-size: 2.0em;
+    color: white;
+  }
+  .navbar-toggle {
+    .icon-bar {
+      background-color: #FFF;
+    }
+  }
+  form>div {
+    margin: 0;
+  }
+  button {
+    font-size: 0.8em;
+  }
+}
+
+.home-page-left {
+  padding-top: 150px;
```

Comment:
I don't think we are ever going to get out of having to use padding and margins on some level to make things work right.